environment.yml

```diff
@@ -6,6 +6,8 @@ dependencies:
 - pip
 - python=3.9.*
 - libgdal=3.4.*
+- geos
```

Comment:
@loicdtx the addition of geos doesn't hurt, but I'm sure it's already a dependency of libgdal.